uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -374,15 +376,54 @@ sub determineSubjectID {
                             $scannerID,
                             $this->{dbhr}
                         );
+                        
+    #==============================================#
+    # Retrieve the candidate's project ID (if any) #
+    # and session's subproject ID                  #
+    #==============================================#
+    (my $query = <<QUERY) =~ s/\n/ /gm;
+    SELECT
+      c.ProjectID, s.SubprojectID
+    FROM
+      session s 
+    JOIN candidate c USING(CandID)
```

Comment:
missing space between USING and (

Reply:
I think this is valid SQL syntax. I assume the space is for clarity?

Reply:
It is valid SQL. It's just that my OCD misses the space ;)

Reply:
Ok